torch/lib/THNN/generic/THNN.h

```diff
@@ -48,7 +48,7 @@ TH_API void THNN_(ClassNLLCriterion_updateOutput)(
           bool sizeAverage,            // if true, the loss will be normalized by batch size and class weights
           THTensor *weights,           // [OPTIONAL] class weights
           THTensor *total_weight,      // [BUFFER]
-          long ignore_index);          // target index to ignore (loss = 0, gradInput = 0)
+          int64_t ignore_index);          // target index to ignore (loss = 0, gradInput = 0)
```

Comment:
nit: the comment is not aligned anymore with the other comments. Not important, but mentioning anyway